### PR TITLE
Update Readme to reference to updated docs

### PR DESCRIPTION
**Pull request by the-dagger**
@mananwason please check and merge.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -6,122 +6,14 @@
...
-
-6. To Build the app, go to *Build>Make Project* (or alternatively press the Make Project icon in the toolbar).
-
-7. If the app was built succesfully, you can test it by running it on either a real device or an emulated one by going to *Run>Run 'app'* or presing the Run icon in the toolbar.
-
-###Video Walkthrough
-[![FOSSASIA](https://img.youtube.com/vi/n5G4yw3t--U/0.jpg)](https://www.youtube.com/watch?v=n5G4yw3t--U)
-
-### Screenshots  
-![alt-tag](android/screenshots/ss2.PNG)
```
Comment (mananwason) [inline README.md]: Please Leave the screenshots and video here only. We need people who visit the repo see them instantly

Comment (the-dagger) [inline README.md]: Ok, will do it.
But I'll add updated screenshots.
The current ones don't look that good.
So you can go ahead and merge this and later tonight I'll be adding new screenshots

Comment (mananwason) [inline README.md]: There's no point in merging this and then create another PR to add screenshots again. You can the new screenshots in the same PR and then we can merge

Comment (the-dagger) [inline README.md]: Ok, I'll do it then.

**Comment by the-dagger** [comment]
@mananwason I have added the screenshots and video walkthrough

**Comment by mananwason** [comment]
LGTM merging 👍

**Pull request merged**